corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -107,61 +106,61 @@ hqDefine('accounting/js/pricing_table', [
                     message = _.template(gettext(
                         "<p>All CommCare subscriptions require a 30 day minimum commitment.</p>" +
                         "<p>Continuing ahead will allow you to schedule your current <%- oldPlan %> " +
-                        "Edition Plan subscription to be paused on <%= dateHtml %>.</p>" +
+                        "Edition Plan subscription to be paused on <strong> <%- date %> </strong></p>" +
                         "<p>If you have questions or if you would like to speak to us about your subscription, " +
-                        "please reach out to <%= emailHtml %>.</p>"
+                        "please reach out to <a href='mailto: <%- invoicingContact %>'><%- invoicingContact %></a>.</p>"
```

Comment:
Was it not possible to sanitize `dateHtml` and `emailHtml` prior to this? I'm assuming `gettext` translates this string, and I don't think it's ideal to ask translators to contribute markup, when possible.

Reply:
I think dealing with HTML is a pretty normal part of translating and Transifex has a decent UI that collapses markup into little non-editable labels. @mkangia might have a more nuanced opinion.

One thing I'm not sure of is whether or not Transifex recognizes the underscore syntax as markup - Manish, do you know?

Reply:
Assuming we have the choice to expose HTML to translators or hide it, what is the advantage to exposing it? I feel what this was prior was more straight-forward with its intent -- the translators get to control the text before and after, but we're in charge of how email addresses are formatted.

Reply:
> the translators get to control the text before and after, but we're in charge of how email addresses are formatted.

I see your point here @mjriley and I think it would make sense to sanitize the variables in JS and then passing in them to underscore by `<%=`
I am merging this for now and would be raising a follow-up PR for the same after the pentest.